lib/tooling/clang-tidy-tool.ts

```diff
@@ -75,14 +87,14 @@ export class ClangTidyTool extends BaseTool {
         // TODO: do we want compile_flags.txt rather than prefixing everything with -extra-arg=
         await fs.writeFile(path.join(dir, 'compile_flags.txt'), compileFlags.join('\n'));
         const result = await super.runTool(compilationInfo, sourcefile, args);
-        result.sourcechanged = false;
+        result.sourceChanged = false;
```

Comment:
We can't make this change here without also updating code in the front-end that reads `sourcechanged`

Reply:
gotcha

Reply:
for simplicity, we can just leave it sourcechanged for now? unless you think it'd be better to change

Reply:
Yes, let's leave it as sourcechanged/newsource for now